build.gradle

```diff
@@ -89,3 +97,13 @@ subprojects {
         systemProperty 'recreate', System.getProperty('recreate', 'false')
     }
 }
+
+nexusPublishing {
```

Comment:
Does this need to live in the root build.gradle or can we put it in the `publish-jar.gradle` (for easy excavation?)

Reply:
Based on https://github.com/gradle-nexus/publish-plugin#applying-the-plugin, its supposed to be applied to the root.